client/src/components/Dialogs/TimerDialog.vue

```diff
@@ -34,23 +20,21 @@
         <span class="mt-2">{{ tabItem.name }}</span>
       </v-tab>
     </v-tabs>
-    <v-tabs-items
-      v-model="tab"
-      class="pt-4 px-4"
-    >
+    <v-tabs-items v-model="tab" class="pt-4 px-4">
       <v-tab-item value="tab-timer">
-        <v-text-field
-          v-model="time"
-          label="Time"
-          outlined
-          disabled
+        <clock-tools-fields
+          :hours.sync="hours"
+          :minutes.sync="minutes"
+          :seconds.sync="seconds"
+          :disabledField="true"
         />
       </v-tab-item>
       <v-tab-item value="tab-countdown">
-        <v-text-field
-          v-model="time"
-          label="Time"
-          outlined
+        <clock-tools-fields
+          :hours.sync="hours"
+          :minutes.sync="minutes"
+          :seconds.sync="seconds"
+          :disabledField="false"
```

Comment:
Again if the prop is falsy you can just skip passing it. For that, it shouldn't be required in props types. And if it's truthy, you don't need to pass "true", the name of the prop would be enough

Reply:
Also, attributes must be hyphenated (disabledField  => disabled-field)